apps/web/components/new-experiment/steps/review-step/review-step.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import * as z from "zod";
+
+import type { CreateExperimentBody } from "@repo/api";
+import { useTranslation } from "@repo/i18n";
+import { WizardStepButtons } from "@repo/ui/components";
+import type { WizardStepProps } from "@repo/ui/components";
+
+import {
+  detailsSchema,
+  locationsSchema,
+  membersVisibilitySchema,
+  protocolsSchema,
+} from "../form-step";
+import { DetailsSection } from "./details-section";
+import { LocationsSection } from "./locations-section";
+import { MembersVisibilitySection } from "./members-visibility-section";
+import { ProtocolsSection } from "./protocols-section";
+
+// For the final review step validate the combined schema of all previous steps
+export const reviewSchema = z.object({
+  ...detailsSchema.shape,
+  ...membersVisibilitySchema.shape,
+  ...protocolsSchema.shape,
+  ...locationsSchema.shape,
+});
+
+export function ReviewStep({
+  form,
+  onPrevious,
+  onNext,
+  goToStep,
+  stepIndex,
+  totalSteps,
+  isSubmitting = false,
+}: WizardStepProps<CreateExperimentBody>) {
+  const formData = form.getValues();
+  const { t } = useTranslation();
+
+  return (
+    <div className="mx-auto space-y-8">
+      {/* Header Section */}
+      <div className="space-y-2">
+        <h2 className="text-lg font-medium tracking-tight">
+          {t("experiments.reviewYourExperiment")}
+        </h2>
+        <p className="text-muted-foreground text-sm">{t("experiments.reviewAllDetails")}</p>
+      </div>
+
+      <div className="grid gap-6 md:grid-cols-2">
+        {/* Details - full width */}
+        <div className="md:col-span-2">
+          <DetailsSection
+            formData={formData}
+            onEdit={() => (goToStep as (index: number) => void)(0)}
+          />
+        </div>
+
+        {/* Members - full width */}
+        <div className="md:col-span-2">
+          <MembersVisibilitySection
+            formData={formData}
+            onEdit={() => (goToStep as (index: number) => void)(1)}
```

Comment:
This can't be right from a syntax perspective goToStep should be typed right in the props so you dont need to cast it